config.json

```diff
@@ -1,8 +1,8 @@
 {
   "formatVersion": 1,
   "menu": {
-    "Config.FINDOLOGICSearchTab": {
-      "label": "Config.FINDOLOGICSearchTab",
+    "Config.FindologicSearchTab": {
```

Comment:
Have you checked if Plentymarkets manages to save these values? Otherwise upgrading would result in all configuration to be removed.

Reply:
Yes click tested and all values are set as before, all container connections are there, nothing changed.
![Screen Shot 2020-04-28 at 17 58 31](https://user-images.githubusercontent.com/511559/80509731-146e2680-897a-11ea-8053-f0f761c4de65.png)
![Screen Shot 2020-04-28 at 17 59 15](https://user-images.githubusercontent.com/511559/80509740-16d08080-897a-11ea-979f-5c6169959e9c.png)